src/util.rs

```diff
@@ -41,7 +42,12 @@ pub fn fzf_helper(now: Epoch, mut dirs: Vec<Dir>) -> Result<Option<String>> {
         dir.rank = dir.get_frecency(now);
     }
 
-    dirs.sort_by_key(|dir| std::cmp::Reverse(dir.rank as i64));
+    dirs.sort_unstable_by(|dir1, dir2| {
+        dir1.rank
+            .partial_cmp(&dir2.rank)
+            .unwrap_or(Ordering::Equal)
```

Comment:
I'm not 100% satisfied with the `.unwrap_or(Ordering::Equal)`, but we aren't expecting any incomparable values, and it's too trivial an issue to stop execution and throw an error. We do want to handle incomparable `f64` values elsewhere, though, presumably with `is_finite`, to ensure that they don't stick around in the database forever.

Reply:
Do we need to handle incomparable `f64`s if we default to a `MAXAGE` of `1000.0`, clamp frecency to `9999.0`, and anything below `1.0` gets removed?